Python/Product/TestAdapter/TestAnalyzer.cs

```diff
@@ -157,13 +157,28 @@ LocationInfo sourceLocation
             }
         }
 
+        /// <summary>
+        /// Get Test Case Members for a class.  If the class has 'test*' tests 
+        /// return those.  If there aren't any 'test*' tests return (if one at 
+        /// all) the runTest overridden method
+        /// </summary>
         private static IEnumerable<KeyValuePair<string, IAnalysisSet>> GetTestCaseMembers(
             IPythonProjectEntry entry,
             AnalysisValue classValue
         ) {
-            return classValue.GetAllMembers(entry.Analysis.InterpreterContext)
+            var tests = classValue.GetAllMembers(entry.Analysis.InterpreterContext)
                 .Where(v => v.Key.StartsWith("test"))
                 .Where(v => v.Value.Any(m => m.MemberType == PythonMemberType.Function || m.MemberType == PythonMemberType.Method));
+
+            var runTest = classValue.GetAllMembers(entry.Analysis.InterpreterContext)
+                .Where(v => v.Key.Equals("runTest"))
+                .Where(v => v.Value.Any(m => m.MemberType == PythonMemberType.Function || m.MemberType == PythonMemberType.Method));

```

Comment:
This is also repeated now. Query for member type first and stash that away, then do name queries on top of that.